target/linux/ar71xx/image/Makefile

```diff
@@ -17,6 +17,7 @@ include ./tp-link.mk
 include ./ubnt.mk
 include ./generic.mk
 include ./legacy-devices.mk
+include ./senao.mk
```

Comment:
A dump of the MTDs has been prepared for you, please send your e-mail address to por~at~xs4all.nl so that I can can send you the dump directly.